lib/param_estimation.py

```diff
@@ -0,0 +1,21 @@
+import json
+import yaml
+from lib.scylla_cloud import get_cloud_instance, is_ec2
+
+def estimate_streaming_bandwidth():
+    cloud_instance = get_cloud_instance()
+    net_bw = 0
+    if is_ec2():
+        instance_type = cloud_instance.instancetype
+        with open('/opt/scylladb/scylla-machine-image/aws_net_params.json') as f:
+            netinfo = json.load(f)
+            instance_info = [info for info in netinfo if info[0] == instance_type]
+            if len(instance_info) != 0:
+                net_bw = int(instance_info[0][2] * 1024 * 1024 * 1024) #GBPS
+    # TODO: other clouds
+
+    if net_bw == 0:
+        return 0
```

Comment:
```suggestion
```
The formula below results with the same value (zero) when `net_be == 0`.
No need a special case here.

Reply:
True. Removed disk part, so... Done